### DISCUSSION TITLE
위키 문서 제목 몇 가지가 분명하지 않아 변경

### DISCUSSION THREAD

**Discussion by greyHairChooseLife** [wanted-pre-onboarding-side-project/habit-tracker#32] [Suggestion]
위키 문서의 제목이 내용과 정확히 일치하지 않아서, 작성에 많이 관여한 제가 봐도 헷갈리더라고요.

문서 제목 몇 가지를 아래와 같이 변경하면 어떨까요?

1. [Merge 컨벤션](https://github.com/wanted-pre-onboarding-side-project/habit-tracker/wiki/Merge-%EC%BB%A8%EB%B2%A4%EC%85%98) :point_right: **upstream/merge commit message**
    : 로컬에서의 merge와는 상관 없는데다가, 안내 되고 있는 내용이 오직 커밋 메시지에 관련됩니다. 너무 많은 규칙들에 압도 되지 않도록 범위를 좁혀서(정확히) 표현합니다.

2. [PR review merge 규칙](https://github.com/wanted-pre-onboarding-side-project/habit-tracker/wiki/PR---review---merge-%EA%B7%9C%EC%B9%99) :point_right: **pr review merge PROCESS**
    : 이 문서는 PR-review-merge가 이루어지는 일련의 과정을 파악하는 용도입니다. 규칙이라고 하니까 PR이나 review를 할 때마다 숙지해야 할 규칙이 있는 건지 혼란을 줄 우려가 있어 보입니다.

**Comment by jiyeon2** [comment]
제시해주신 제목이 더 정확하네요~

개인적으로 제목이 전부 영어면 잘 안읽혀서
1. upstream/merge 커밋 메시지 컨벤션
2. pr 리뷰 과정
정도면 좋겠습니다만 상연님이 별로인거같은데.. 라는 생각이 0.0001초라도 드셨다면
저는 제시해주신 제목 그대로 변경하는 데 1000000% 찬성합니다

**Reply by greyHairChooseLife** [reply]
맞아요, 저도 한글이 더 잘 읽힙니다!

2번 항목은 내용에 merge까지 있으니까 아래 처럼 전부 써주면 어떠실까요?

1. upstream/merge 커밋 메시지 컨벤션
2. PR 리뷰와 merge 과정

**Reply by jiyeon2** [reply]
오 좋습니다 :+1:
